html-aam/index.html

```diff
@@ -424,7 +424,7 @@ <h4 id="el-abbr">`abbr`</h4>
             </tr>
             <tr>
               <th><a data-cite="core-aam-1.2/#roleMappingComputedRole">Computed Role</a></th>
-              <td class="role-computed"><div class="general">html-abbr</div></td>
+              <td class="role-computed"><div class="general">`html-abbr`</div></td>
```

Comment:
FYI neither `.role-computed` nor `.general` do anything (in case someone wants to clean them up some time).

Reply:
=> https://github.com/w3c/aria/issues/2621